src/window/window.rs

```diff
@@ -324,7 +324,7 @@ impl Window {
 
         window.make_current();
 
-        verify!(gl::load_with(|name| window.get_proc_address(name)));
+        verify!(gl::load_with(|name| mem::transmute(window.get_proc_address(name))));
```

Comment:
You don't need a transmute here. `window.get_proc_address(name) as *const _` should suffice.